css/style.css

```diff
@@ -0,0 +1,5 @@
+* {
+margin: 0;
+padding: 0;
+box-sizing: border-box;
```

Comment:
This code has some indentation errors,I would suggest you install the linter files and run the necessary commands to have a cleaner code👍